src/pages/Lecture/LectureSideBar.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import LectureSideContent from './LectureSideContent';
+
+function LectureSideBar() {
+  const [open, setOpen] = useState(false);
+
+  const handleToggle = () => setOpen(!open);
```

Comment:
```suggestion
  const handleToggle = () => setOpen(prev => !prev);
```
이렇게 바꾸면 외부 값에 의존 안하고 같은 동작을 할거 같네요